test/test.ml

```diff
@@ -1,23 +1,26 @@
 type ava
-type exe
+type t
 
-external t:ava = "ava" [@@bs.module]
+external ava: ava = "ava" [@@bs.module]
```

Comment:
I don't think there's any need to expose this. It could just be an internal Js.t used directly by `test` and such, e.g. `let test name f = ava##test name f`.